test/integration/e2e/admin-api/patchPayId.test.ts

```diff
@@ -0,0 +1,281 @@
+import HttpStatus from '@xpring-eng/http-status'
+import * as request from 'supertest'
+import 'mocha'
```

Comment:
nit: Have the mocha import up top by itself.

```suggestion
import 'mocha'

import HttpStatus from '@xpring-eng/http-status'
import * as request from 'supertest'
```